### DISCUSSION TITLE
random walls

### DISCUSSION THREAD

**Discussion by rishav12s** [Matt-FTW/dotfiles#38] [Q&A]
how to disable random walls ?

tried moving the script to a different location from `~/.config/hypr/scripts/`

and where are the scripts getting executed are mentioned cant seem to find it

**Comment by rishav12s** [comment]
ahh got it pyprland is responsible for this lol

**Reply by Matt-FTW** [reply]
I've added a new section in the tips page on the docs letting everyone know how to disable it. Thanks for letting me know about this haha :)
